src/State/ViewRequests/sagas.test.js

```diff
@@ -0,0 +1,158 @@
+import { runSaga } from 'redux-saga';
```

Comment:
May want to have a test directory of some sort

Reply:
Abby this is awesome. 
Yeah agree Jake we might want test folder.

Reply:
test folder added!